### PR TITLE
Customize devise pages

### PR DESCRIPTION
**Pull request by guragt**
1. Log in page
Screenshot 2020-10-15 at 13 11 19
Screenshot 2020-10-15 at 13 12 28

2. Forgot password page
Screenshot 2020-10-15 at 13 13 36

3. Change password page
Screenshot 2020-10-15 at 13 15 18

4. Edit page
Screenshot 2020-10-15 at 17 33 01

### CODE REVIEW

File: app/views/devise/passwords/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <%= simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
+
+      <%= f.input :reset_password_token, as: :hidden %>
+      <div class="invalid-feedback">
+        <%= f.full_error :reset_password_token %>
+      </div>
+
+      <div class="form-inputs">
+        <%= f.input :password,
+                    label: "New password" %>
```
Comment (Snick555) [inline app/views/devise/passwords/edit.html.erb]: make all text to be localized

Comment (guragt) [reply app/views/devise/passwords/edit.html.erb]: Done

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -1,43 +1,23 @@
...
+      <div class="form-actions">
+        <%= f.button :submit, t("views.registrations.edit.submit"), class: "btn btn-primary" %>
+      </div>
     <% end %>
-  </div>
 
-  <div class="field">
-    <%= f.label :password_confirmation %><br />
-    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    <%= link_to t("views.registrations.edit.remove"), registration_path(resource_name), class: "btn btn-danger wide-button", data: { confirm: "Are you sure?" }, method: :delete %>
```
Comment (Snick555) [inline app/views/devise/registrations/edit.html.erb]: missed localization

File: app/views/devise/passwords/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        <%= f.full_error :reset_password_token %>
+      </div>
+
+      <div class="form-inputs">
+        <%= f.input :password %>
+        <%= f.input :password_confirmation %>
+      </div>
+
+      <div class="form-actions">
+        <%= f.button :submit, t("views.passwords.edit.submit"), class: "btn btn-primary" %>
```
Comment (Snick555) [inline app/views/devise/passwords/edit.html.erb]: read this https://guides.rubyonrails.org/i18n.html#lazy-lookup

**Review by Snick555 — APPROVED** [review]

**Pull request merged**